package.json

```diff
@@ -15,16 +15,16 @@
     "url": "https://github.com/atlassian/react-beautiful-dnd/issues"
   },
   "main": "lib/index.js",
-  "files": [
-    "/lib"
-  ],
+  "files": ["/lib", "/dist"],
```

Comment:
Would be good to also add a `browser` field that maps to `dist/index.js` (it will be minified by bundlers, however unpkg won't serve the min file).